pages/oss.md

```diff
@@ -37,13 +37,13 @@ Communities are critically important to the long term viability of open source p
 
 <li>Leveraging Existing Communities – Whenever possible, custom code released to the public as OSS should be incorporated into existing communities of practice that are self-sustaining. For example, there are already existing communities for electronic health records and geospatial software.<sup id="fnr37"><a href="#fn37">37</a></sup> Government agencies should only develop their own communities when existing communities do not satisfy their needs.</li>
 
-<li>Open Development – Software that is custom-developed for or by covered agencies should, to the extent possible and appropriate, be developed in the open. Open development practices provide an environment in which open source code can flourish and repurposed. This principle, as well as the principle for "Releasing Code" below, shall include the distribution of a minimum viable product as open source code, engaging the public before official release,<sup id="fnr38"><a href="#fn38">38</a></sup> and drawing upon the public's knowledge for bug fixes, algorithmic optimization, and other improvements to the project.</li>
+<li>Open Development – Software that is custom-developed for or by covered agencies should, to the extent possible and appropriate, be developed in the open from the start. Open development practices provide an environment in which open source code can flourish and repurposed. This principle, as well as the principle for "Releasing Code" below, shall include engaging the public on an ongoing basis,<sup id="fnr38"><a href="#fn38">38</a></sup> and drawing upon the public's knowledge for bug fixes and other improvements to the project.</li>
```

Comment:
The diff doesn't appear well here for some reason, but the change is adding the bolded text:

> agencies should, to the extent possible and appropriate, be developed in the open **from the start**

It is far easier to move to open development the earlier in the process it is done, and that project will also benefit from things like external review/contributions sooner than later. Getting early feedback can help guide architectural decisions, which are difficult to reverse once made. Encouraging open development leads to better software, which leads to better government.